src/test/resources/features/registrationEmailLarissa.feature

```diff
@@ -0,0 +1,114 @@
+@registrationEmailLarissa
+  Feature: Registration Email
+
+    Scenario: Registration Email_Verify alphanumeric and special char
+      Given I go to "registration" page
+      When I type "larissaomalley1!@gmail.com" into email field
+      Then I type "Mickey" into First Name field
+      Then I type "Mouse" into the Last Name field
+      Then I type "Jun02" into groupCode field
+      Then I enter "12345" as password
+      Then I enter "12345" as Confirm Password
+      Then I click button "Register Me"
```

Comment:
Please add step "Then confirmation message "You have been Registered." should be displayed